### PR TITLE
Replace old editor

### PR DESCRIPTION
**Pull request by IagoLast**
- Missing tutorial/debugging steps

### CODE REVIEW

**Comment by Jesus89** [comment]
I think it should be very useful to keep the loader (spinner)

**Comment by makella** [comment]
By default, the styling edtior is too small and it isn't super intuitive how to make it taller.
screen shot 2018-06-20 at 9 08 41 am

Can we increase the default height and also the font size maybe by 1px?

screen shot 2018-06-20 at 9 10 48 am

cc @arianaescobar

**Comment by makella** [comment]
can we put the information for zoom and center back at the bottom of the map? and have the feature count on the information panel on the left? (just a thought)

also, the formatting for the center and zoom needs a space between the `center [ , ] , zoom ....`
screen shot 2018-06-20 at 9 15 41 am

File: examples/editor/index.html [hunk 1]
```diff
@@ -1,177 +1,89 @@
...
 <head>
-    <meta charset='utf-8' />
-    <title>Display a map</title>
-    <meta name='viewport' content='initial-scale=1,maximum-scale=1,user-scalable=no' />
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js" integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="
-        crossorigin="anonymous"></script>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CARTO-VL EDITOR</title>
```
Comment (davidmanzanares) [inline examples/editor/index.html]: `CARTO VL` not `CARTO-VL`

**Comment by IagoLast** [comment]
> The debugging steps are broken since you are pointing to another dataset with other column names, please use BC.

eing?

**Comment by davidmanzanares** [comment]
Try the buttons, it's broken

**Comment by IagoLast** [comment]
what is the exact meaning of broken?

I see this:

[https://gyazo.com/11326c7983e1c039132383bea8cc6da2](https://gyazo.com/11326c7983e1c039132383bea8cc6da2)

**Comment by IagoLast** [comment]
ok fixed

**Comment by davidmanzanares** [comment]
buggy-gen-example

The generated .html show unwanted scrolls :slightly_frowning_face:

**Pull request merged**